cloudferry_devlab/cloudferry_devlab/tests/cleanup.py

```diff
@@ -80,6 +80,12 @@ def clean_volumes(self):
                 self.get_volume_id(volume.display_name))
             LOG.info('Volume "%s" has been deleted', volume.display_name)
 
+    def delete_volume_by_name_from_db(self, vol_name, position):
+        cinder_mysql = self.mysql_connector('cinder', position)
+        sql_vol_delete_by_name = "DELETE FROM volumes WHERE"\
+            " display_name='{vol_name}'".format(vol_name=vol_name)
```

Comment:
I'd recommend removing based on volume ID instead of volume name because it's more reliable